.github/prlint.json

```
@@ -1,8 +1,8 @@
 {
   "title": [
     {
-      "pattern": "^(feature|bug|minor|docs|DX):\\s.+\\S$",
-      "message": "Your title needs to match the convention: use '(feature|bug|minor|docs|DX):' prefix."
+      "pattern": "^(feat(ure)?|bug|chore|CI|docs|DX|refactor|test)(\\([a-zA-Z0-9]+\\))?:\\s{1}.+\\S$",
```

```suggestion
      "pattern": "^(feat(ure)?|bug|chore|CI|docs|DX|refactor|test)(\\([a-zA-Z0-9]+\\))?:\\s.+\\S$",
```

what for you added `{1}` ? like, the next element is `.`, which allows for any amount of \s directly after first space, anyway

---

You're right, but instead of removing `{1}` I just added `\\S` there to enforce non-whitespace char after single space.